ptcg_cockatrice.xml

```diff
@@ -1189,12 +1189,12 @@ Whenever Cloyster becomes tapped, it deals 1 damage to each of up to three targe
 			<name>Gastly</name>
 			<set picURL="https://raw.githubusercontent.com/brooks42/ptcg/main/cards/Gastly.png" rarity="common">PTCG</set>
 			<color>B</color>
-			<manacost>1B</manacost>
+			<manacost>B</manacost>
 			<type>Pokémon - Ghost Poison</type>
 			<pt>1/1</pt>
 			<tablerow>2</tablerow>
-			<text>{B}{B}: Evolve
-Protection from Ground and Fighting</text>
+			<text>{B}: Evolve
+Protection from Fighting and Normal</text>
```

Comment:
Says "Protection from fighting and normal" but the card art has "from fighting and _from_ normal"